File: CodeForces/CppCodeForces/69a-YoungPhysicist.cpp
//106993934	Feb/09/2021 20:18UTC+5.5	shamanthmuroor	69A - Young Physicist	GNU C++17	Accepted	62 ms	0 KB

//Question 69a: Young Physicist

#include <iostream>

using namespace std;

int main() {
    int n, x, y, z;
    int xv=0, yv=0, zv=0;
    cin >> n;
    for(int i=0; i<n; i++) {
        cin >> x >> y >> z;
        xv += x;
        yv += y;
        zv += z;
    }
    if(xv == 0 && yv == 0 && zv==0) {
        cout << "YES";
    } 
    else 
        cout << "NO";
    return 0;
}

File: CodeForces/69a-YoungPhysicist.py
#106992848	Feb/09/2021 20:07UTC+5.5	shamanthmuroor	69A - Young Physicist	Python 3	Accepted	154 ms	0 KB

""" Question 69a: Young Physicist """

n = int(input())
xvect = yvect = zvect = 0
for i in range(n):
    x, y, z = map(int, input().split())
    xvect += x
    yvect += y
    zvect += z
if xvect == 0 and yvect == 0 and zvect == 0:
    print("YES")
else:
    print("NO")